db/migrate/20181108195546_migrate_miq_policy_towhat_to_resource_type.rb

```diff
@@ -0,0 +1,24 @@
+class MigrateMiqPolicyTowhatToResourceType < ActiveRecord::Migration[5.0]
+
+  class MiqPolicy < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    say_with_time("Moving MiqPolicy towhat to resource type") do
+      MiqPolicy.all.each do |policy|
```

Comment:
I dont think there's too many, but maybe we want find_each

Reply:
or eventually one sql query `MiqPolicy.update_all('resource_type = resource_type')`